rollup.config.js

```diff
@@ -25,12 +25,12 @@ export default [
 			{
 				file: packageJson.main,
 				format: "cjs",
-				sourcemap: true,
+				sourcemap: false
```

Comment:
We should include source maps, why'd you turn them off?

Reply:
I was playing with the bundle size when I was trying to include the giant mermaid library, but also sourcemaps weren't working for me and still aren't after I turned them back on. Might need to spend a sec trying to figure out why. I suspect something happened when I updated the various rollup packages maybe?